package/src/components/MessageInput/UploadProgressIndicator.tsx

```diff
@@ -99,6 +107,17 @@ export const UploadProgressIndicator: React.FC<UploadProgressIndicatorProps> = (
             />
           </TouchableOpacity>
         )}
+        {type === ProgressIndicatorTypes.NOT_SUPPORTED && (
+          <>
+            {/* <Text style={{ color: 'black', textAlign: 'center' }}>File type not supported</Text>
+            <Flag
+              height={18}
+              pathFill={white_smoke}
+              testID='retry-upload-progress-indicator'
+              width={18}
+            /> */}
+          </>
+        )}
       </View>
```

Comment:
Let's see if returning early from the component works as well - then we can drop this :)